packages/harpguru-core/src/components/new-harpstrata-state-callback/hooks/use-derive-activedegreeids/use-derive-activedegreeids.ts

```diff
@@ -0,0 +1,15 @@
+import type { ActiveDegreeIds, HarpStrata } from 'harpstrata'
+
+import { deriveActiveDegreeIds } from '../../utils'
+import { compareActiveIds } from '../../../../utils/compare-active-ids'
+
+export const useDeriveActiveDegreeIds = (
+  newHarpStrata: HarpStrata,
+  prevActiveDegreeIds: ActiveDegreeIds,
+  setActiveDegreeIds: (arg0: ActiveDegreeIds) => void
```

Comment:
Remove `ActiveDegreeIds` from `harpstrata` exports and use `ReadonlyArray<DegreeIds>`.

Reply:
Raised as techdebt in https://github.com/js-jslog/harpguru/issues/120#issuecomment-938452028